diff -urN florence-0.5.0/src/xkeyboard.c florence-0.5.0-patched/src/xkeyboard.c
--- florence-0.5.0/src/xkeyboard.c	2010-11-29 00:59:13.000000000 +1000
+++ florence-0.5.0-patched/src/xkeyboard.c	2011-09-10 11:48:26.088946847 +1100
@@ -173,7 +173,8 @@
 				xkeyboard_layout(xkeyboard);
 			}
 			flo_debug(_("XKB state notify event received"));
-			xkeyboard->event_cb(xkeyboard->user_data);
+			if (xkeyboard->user_data)
+			    xkeyboard->event_cb(xkeyboard->user_data);
 		}
 	}
 	return GDK_FILTER_CONTINUE;
